src/test/java/fixture/MenuFixture.java

```diff
@@ -0,0 +1,56 @@
+package fixture;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+
+public class MenuFixture {
+	private static final String VALID_MENU_GROUP_NAME = "점심특선";
+	private static final String VALID_MENU_NAME = "버거세트";
+	private static final int VALID_MENU_PRODUCT_QUANTITY = 2;
+	private static final BigDecimal VALID_MENU_PRICE = new BigDecimal("17.98");
+
+	public static MenuGroup createValidMenuGroup() {
+		UUID menuGroupId = UUID.randomUUID();
+		MenuGroup validMenuGroup = new MenuGroup();
+		validMenuGroup.setId(menuGroupId);
```

Comment:
Menu 생성 로직과 동일하게 menuGroupId는 굳이 변수로 추출할 필요는 없을 것 같네요. 🙂

```suggestion
		MenuGroup validMenuGroup = new MenuGroup();
		validMenuGroup.setId(UUID.randomUUID());
```

Reply:
수정 반영하겠습니다.